samples/go/face-merge/main.go

```diff
@@ -117,29 +134,27 @@ func filterExtraFields(photo types.Struct) types.Struct {
 }
 
 func getMergedFaces(photo types.Struct) types.Set {
-	facesRect, rectExists := photo.MaybeGet("facesRect")
-	facesCentered, centerExists := photo.MaybeGet("facesCentered")
+	facesRect := photo.Get("facesRect")
+	facesCentered := photo.Get("facesCentered")
 	faceSet := types.NewSet()
 
-	if rectExists && centerExists {
-		facesCentered.(types.Set).Iter(func(faceCentered types.Value) bool {
-			facesRect.(types.Set).Iter(func(faceRect types.Value) bool {
-				facePts := getFacePoints(faceRect.(types.Struct), faceCentered.(types.Struct))
-				if centeredFaceAligned(facePts) {
-					faceSet = faceSet.Insert(types.NewStruct("", types.StructData{
-						"x":    facePts.x,
-						"y":    facePts.y,
-						"w":    facePts.w,
-						"h":    facePts.h,
-						"name": facePts.name,
-					}))
-					return true
-				}
-				return false
-			})
+	facesCentered.(types.Set).Iter(func(faceCentered types.Value) bool {
+		facesRect.(types.Set).Iter(func(faceRect types.Value) bool {
+			facePts := getFacePoints(faceRect.(types.Struct), faceCentered.(types.Struct))
+			if centeredFaceAligned(facePts) {
+				faceSet = faceSet.Insert(types.NewStruct("", types.StructData{
+					"x":    facePts.x,
+					"y":    facePts.y,
+					"w":    facePts.w,
+					"h":    facePts.h,
+					"name": facePts.name,
+				}))
+				return true
```

Comment:
For this signature we usually name the return value, like this, so that it's clear what the meaning is while reading:

```
func (faceRect types.Value) (stop bool) {

}
```

In that case, at this line you'd do:

`stop = true`

and the only return would be at line 154, and it'd just be:

`return`